ksql-parser/src/test/java/io/confluent/ksql/parser/KsqlParserTestUtil.java

```diff
@@ -42,8 +43,12 @@ private KsqlParserTestUtil() {
   }
 
   public static List<PreparedStatement<?>> buildAst(final String sql, final MetaStore metaStore) {
-    return KSQL_PARSER.parse(sql).stream()
+    return parse(sql).stream()
```

Comment:
Unrelated to your PR, but why are there two `KsqlParserTestUtil`s (one in `ksql-engine` and the other in `ksql-parser`) that are both public and nearly identical...?

Reply:
Probably a dependency issue, so there are two issues. 

It might be possible to have one set of tests depend on the other set of tests, and hence have a single KsqlParserTestUtil. Or it might be there are reasons why this doesn't work, and hence there are two.